src/components/Tooltip/EducationalTooltip/BaseEducationalTooltip.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {memo, useEffect, useRef} from 'react';
+import type {LayoutEvent} from 'react-native';
+import GenericTooltip from '@components/Tooltip/GenericTooltip';
+import type TooltipProps from '@components/Tooltip/types';
+import getBounds from './getBounds';
+
+/**
+ * A component used to wrap an element intended for displaying a tooltip.
+ * This tooltip would show immediately without user's interaction and hide after 5 seconds.
+ */
+function BaseEducationalTooltip({children, ...props}: TooltipProps) {
+    const hideTooltipRef = useRef<() => void>();
+
+    // Automatically hide tooltip after 5 seconds
+    useEffect(() => {
+        if (!hideTooltipRef.current) {
+            return;
+        }
+
+        const intervalID = setInterval(hideTooltipRef.current, 5000);
+        return () => {
+            clearInterval(intervalID);
+        };
```

Comment:
Oh I didn't know you could return a cleanup function in useEffect, I learned something new!